website/addons/s3/static/tests/s3NodeConfig.test.js

```diff
@@ -385,15 +385,9 @@ describe('s3NodeConfigViewModel', () => {
                 var vm = new s3NodeConfigVM('/api/v1/12345/s3/settings/', '', '/12345');
                 vm.updateFromData()
                     .always(function() {
-                        var promise = vm.createCredentials();
-                        assert.isTrue(vm.creatingCredentials());
-                        assert.isFalse(vm.userHasAuth());
-                        server.respond();
-                        promise.always(function() {
-                            assert.isFalse(vm.creatingCredentials());
-                            assert.isTrue(vm.userHasAuth());
-                            done();
-                        });
+                        assert.isFalse(vm.createCredentials());
+                        assert.isFalse(vm.creatingCredentials());
+                        done();
                     });
```

Comment:
@sloria I add a simple test for the case where secret_key and access_key are empty string.

 However, I still feel confusing about the code in original test case.  A surprise for me is that when I bring the old code back, I can't find these test code in webpacked test file. It seems like it is not being executed.  And also I can't find what value secret_key and access_key are defined.

When doing testing for creating credential,  I have to add the value of secret_key and access_key, so what they should be?